components/kyma-environment-broker/internal/process/steps/lifecycle_manager.go

```diff
@@ -0,0 +1,32 @@
+package steps
+
+import (
+	"fmt"
+	"strings"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+)
+
+// Set common labels for kyma lifecycle manager
+func ApplyLabelsForLM(object client.Object, operation internal.Operation) {
+	l := object.GetLabels()
+	if l == nil {
+		l = make(map[string]string)
+	}
+	l["kyma-project.io/instance-id"] = operation.InstanceID
+	l["kyma-project.io/runtime-id"] = operation.RuntimeID
+	l["kyma-project.io/broker-plan-id"] = operation.ProvisioningParameters.PlanID
+	l["kyma-project.io/global-account-id"] = operation.GlobalAccountID
+	l["operator.kyma-project.io/kyma-name"] = KymaName(operation)
+	object.SetLabels(l)
```

Comment:
potential reuse for this in https://github.com/kyma-project/control-plane/pull/2181